sycl/test-e2e/VirtualMem/basic_access_from_kernel_virtual_mem.cpp

```diff
@@ -0,0 +1,87 @@
+// REQUIRES: aspect-ext_oneapi_virtual_mem
```

Comment:
I think that we should probably add a brief comment describing what test is trying to check, or at least give a reference to the corresponding test plan section with the description for this test case.